scripts/Packages/Package_DFN_QFN/size_definitions/wson.yaml

```diff
@@ -76,6 +76,49 @@ WSON-8-1EP_6x5mm_P1.27mm_EP3.4x4.0mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+WSON-8-1EP_6x5mm_P1.27mm_E3.4x4.3mm:
+  device_type: 'WSON'
+  library: Package_SON
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://www.winbond.com/resource-files/w25q32jv%20revg%2003272018%20plus.pdf (page 68)'
+  ipc_class: 'qfn_pull_back' #'qfn' | 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  #body_size_x_min: 2.85
+  body_size_x:
+    nominal: 6
+    tolerance: 0.1
+  #body_size_x_max: 3.15
+  #body_size_y_min: 2.85
+  body_size_y:
+    nominal: 5
+    tolerance: 0.1
+  #body_size_y_max: 3.15
```

Comment:
Please add the body height parameter as well (for generating 3d models from the same yaml definition files)

```yaml
body_height:
  minimum: 0.70
  nominal: 0.75
  maximum: 0.80
```